DataDog/dd-trace-py
# [tests] run flake8 on all test files

This PR does a few small things... which resulted in a ton of file changes.

- `tox.ini`
  - Run `flake8` on the entire project directory, not just `ddtrace`
  - This means we will now run `flake8` against `tests` and `setup.py`
  - We are also updating the `exclude=` `flake8` config to be the default + `.ddtox`
- `tests/contrib/*`
  - Add `# flake8: noqa` to all files which were failing tests
- `tests/opentracer/*`
  - Add `# flake8: noqa` to all files which were failing tests

We went ahead and fixed a handful of the `flake8` issues in most test files, but decided to just blacklist the specific ones in `tests/contrib/*` and `tests/opentracer/*` since there were a ton of them and it'll help reduce the size of this PR. In the future we should go through one by one and update those files to remove the `# flake8: noqa` from the top of the file.
